source/_integrations/tts.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to set up Text-to-Speech (TTS) with Home Assist
 ha_category:
   - Text-to-speech
 ha_release: 0.35
+ha_iot_class: Cloud Push
```

Comment:
This is an entity component, it cannot be used by itself, but is here for other integration to build upon. Because one cannot use this integration directly, it doesn't have an IoT class.

```suggestion
ha_iot_class: ~
```

Reply:
That makes more sens indeed. 
Thanks!